src/test/regress/expected/insert_select_connection_leak.out

```diff
@@ -23,7 +23,7 @@ SELECT create_distributed_table('target_table', 'a');
 
 (1 row)
 
-INSERT INTO source_table SELECT i, 2 * i FROM generate_series(1, 10000) i;
+INSERT INTO source_table SELECT i, 2 * i FROM generate_series(1, 100) i;
```

Comment:
I have not much context on this test file but I wonder if we were inserting 1000 rows intentionally (for example, to test higher number of connections for connection leak test)

Reply:
I think @pykello said `100` and `10000` are okay for the purpose of this test file.

Reply:
I've also tried that but the running time was very high. 15 seconds, 20 seconds, 42 seconds etc. It doesn't cause a timeout but still takes too much time.